src/runner.py

```diff
@@ -109,46 +109,6 @@ def _run(self, sub_id):
         return True
 
 
-class RunnerSingleJob(Runner):
-    """Runner subclass to execute a single job"""
-
-    def _get_node_from_commit(self, git_commit):
-        nodes = self._api.get_nodes({
-            "revision.commit": git_commit,
-        })
-        return nodes[0] if nodes else None
```

Comment:
I see `kci job` command doesn't have provision to generate job from git commit.
Are you planning to add it?
If so, this should be dropped after adding the provision.

Reply:
Generally speaking, the pipeline scripts aren't command line tools so the trend should be to remove virtually all the code from them and have functions implemented in kernelci-core.  Then we'll have the `kci` command line interface on top of it for manual work, and the pipeline services for automation (and we might implement the service loops in kernelci-core directly eventually but that's a grey area still).

Reply:
Okay.
The only concern I had was to make sure the change is integrated to `kci job` before we drop it from here.
I think we are good to go.